Tools/LyTestTools/tests/integ/sanity_tests.py

```diff
@@ -6,49 +6,86 @@
 A sanity test for the built-in fixtures.
 Launch the windows launcher attached to the currently installed instance.
 """
+# Import any dependencies for the test.
```

Comment:
Nit: Do we need comments explaining what imports are?

Reply:
I think it's reasonable to expect basic python knowledge